src/core/Breadcrumb/BreadcrumbLink.tsx

```diff
@@ -43,7 +46,8 @@ export const BreadcrumbLink = ({
       <Icon
         icon="linkBreadcrumb"
         className={iconClassName}
-        color={withSuomifiDefaultProps(passProps).tokens.colors.blackBase}
+        color={colorValue(passProps)('blackBase')}
```

Comment:
Would it be possible to use e.g. ```passprops.tokens.colors.blackBase.hsl``` instead of the colorValue utility function or would that break modularity somehow? What are the benefits of using a utility instead of directly accessing the values in components?

Reply:
It is also ok to keep this abstraction to create looser coupling with design-system-tokens repo.